### PR TITLE
Add settings to allow easy configuration of the ticket name format

### PR DESCRIPTION
**Pull request by cjflory**

### CODE REVIEW

**Comment by newportt** [comment]
:+1:

File: censeo/settings.py [hunk 1]
```diff
@@ -109,5 +108,14 @@
 LOGIN_REDIRECT_URL = reverse_lazy('meet')
 ACCOUNT_ACTIVATION_DAYS = 3
 
+# Ticket Validation Settings
+TICKET_REGEX = r'LON-\d{4}'
```
Comment (garrypolley) [inline censeo/settings.py:112]: :frowning:   That's really specific.

Comment (cjflory) [inline censeo/settings.py:112]: That's why I made it easily configurable  :tongue:

**Comment by cjflory** [comment]
@garrypolley  Added the CDN fallback I mentioned today

**Pull request merged**